### ISSUE TITLE
Mention `cy.request()` is not affected by `cy.server()`

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress-documentation#2452]
## I'm submitting a...

```
[ ] Bug report
[x] Content update
[ ] Process update (build, deployment, ... )
```

## Type of bug / changes

The Note from cy.request() should also be mentioned on the `cy.server()` doc. https://docs.cypress.io/api/commands/request.html#cy-request-cannot-be-used-to-debug-and

**Comment by CypressCecelia** [comment]
@jennifer-shehane I think there was already a branch for this, did the PR get submitted?

**Comment by jennifer-shehane** [comment]
Sorry, yes. I don't know why I forgot to close this again. Closed in #2453
